tests/components/rfxtrx/test_cover.py

```diff
@@ -55,20 +55,25 @@ async def test_one_cover(hass, rfxtrx):
     )
     await hass.async_block_till_done()
 
-    hass.data[rfxtrx_core.DATA_RFXOBJECT] = rfxtrxmod.Core(
-        "", transport_protocol=rfxtrxmod.DummyTransport
+    assert len(hass.states.async_all()) == 1
+
+    await hass.services.async_call(
+        "cover", "open_cover", {"entity_id": "cover.test"}, blocking=True
+    )
+
+    await hass.services.async_call(
+        "cover", "close_cover", {"entity_id": "cover.test"}, blocking=True
+    )
+
+    await hass.services.async_call(
+        "cover", "stop_cover", {"entity_id": "cover.test"}, blocking=True
     )
 
-    assert 1 == len(rfxtrx_core.RFX_DEVICES)
-    for id in rfxtrx_core.RFX_DEVICES:
-        entity = rfxtrx_core.RFX_DEVICES[id]
-        entity.hass = hass
-        assert entity.signal_repetitions == 1
-        assert not entity.should_fire_event
-        assert not entity.should_poll
-        entity.open_cover()
-        entity.close_cover()
-        entity.stop_cover()
+    assert rfxtrx.transport.send.mock_calls == [
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5802827)